### DISCUSSION TITLE
Styling GeoJSON Simple Style Spec

### DISCUSSION THREAD

**Discussion by geospatialmapmashup** [TerriaJS/terriajs#4989] [Q&A] [answered]
GeoJSON 
https://docs.terria.io/guide/connecting-to-data/catalog-type-details/geojson/
style

An object of style information which will be used instead of the default, but won't override styles set on individual GeoJSON features. Styles follow the SimpleStyle spec: https://github.com/mapbox/simplestyle-spec/tree/master/1.1.0 marker-opacity and numeric values for marker-size are also supported.

I know that GeoJSON.io and other sites can write MapBox Simple Style as part of the GeoJSON
![image](https://user-images.githubusercontent.com/15051730/99099162-05fafb80-25a8-11eb-8afe-3affba8f3e9c.png)
Do you have any example catalogs for us to see the best approaches of handling this
I want to handle this at Catalog Level
unless we really need different features styled differently
![image](https://user-images.githubusercontent.com/15051730/99100219-89691c80-25a9-11eb-8724-36894b8bd82f.png)

**Comment by hardreddata** [comment, marked as answer]
It is basically

```
{
  "catalog": [
    {
      "name": "Point",
      "type": "geojson",
      "url": "https://gist.githubusercontent.com/steve9164/c517308cf52638e1ff3a2992e5aec3d1/raw/b53d846db4e870dca1681332db40962689aa7c35/point.geojson",
      "style": {
        "marker-url": "http://ci.terria.io/master/images/terria_logo.png"
      }
    }
  ]
}
```

Shamelessly lifted from @steve9164 and his [gist](https://gist.githubusercontent.com/steve9164/7cd7099b0495875ee0bf1ff651922095/raw/232fc7f73a685f9bab2288709494769335e932cc/test-point-marker-url.json)

I also found https://raw.githubusercontent.com/TerriaJS/terriajs/master/wwwroot/test/GeoJSON/test-styles.geojson and didn't realise we could do this. Very cool!
